### DISCUSSION TITLE
How to remove background color of active button

### DISCUSSION THREAD

**Discussion by JaLe29** [react-navigation/react-navigation#10855] [Q&A] [answered]
I would like to remove (or change) background color (#E8DEF8) of active button. Which props should I use? I tried almost everything from documentation.

![image](https://user-images.githubusercontent.com/22296816/190848974-98e886a6-09b1-4420-a8a4-b13d13952719.png)

**Comment by r-hirsch** [comment, marked as answer]
TL:DR at the bottom.

Looks like you're using React Native Paper (RNP) V5 themeing with React Navigation. RNP V5 utilises theming based on Material Design 3 (MD3). Take a look [here](https://callstack.github.io/react-native-paper/introducing-v5-with-material-you.html#theming) to see the theming of RNP V5. Then if you look [here](https://m3.material.io/components/navigation-bar/specs) you can see the specification of the Navigation Bar in MD3 and in the table they state the colour of the Active indicator background is _md.sys.color.secondary-container_. Note it is the secondary container colour value. So, when you provide your `theme` to the `PaperProvider` at the top-level of your app you just need to change the `secondaryContainer` field value in the `colors` object in the `theme` prop. If you haven't yet made a combined theme for RNP and React Navigation I would suggest looking [here](https://callstack.github.io/react-native-paper/theming-with-react-navigation.html).

TLDR: Change the `secondaryContainer` field value in `colors` in the `theme` prop you provide to `PaperProvider` and `NavigationContainer`.

**Reply by JaLe29** [reply]
Thank you!

**Reply by r-hirsch** [reply]
You're welcome :) Happy coding
